add the old name (barrier) as a keyword in the desktop file
so it is discoverable in things that launch desktop files

--- a/res/io.github.input_leap.InputLeap.desktop
+++ b/res/io.github.input_leap.InputLeap.desktop
@@ -6,4 +6,4 @@
 Icon=io.github.input_leap.InputLeap
 Terminal=false
 Categories=Network
-Keywords=keyboard;mouse;sharing;network;share;
+Keywords=keyboard;mouse;sharing;network;share;barrier
